pom.xml

```diff
@@ -55,7 +55,7 @@
             <groupId>dev.openfeature</groupId>
             <artifactId>sdk</artifactId>
             <!-- 1.0 <= v < 2.0 -->
-            <version>[1.0,2.0)</version>
+            <version>[1.4.0,2.0)</version>
```

Comment:
This will make `1.4.0` a requirement for everything. I think we want to add this section to just the otel hook

Reply:
hmm, I will check if this is possible

Reply:
Yup, it's possible - https://github.com/open-feature/java-sdk-contrib/pull/332/files#diff-04c08d30909eade5e2f0b110b60da00732703796061a62a38bdb7ae66db549caR40-R46

Updated hook pom to have this depdency